doc/rados/configuration/mclock-config-ref.rst

```diff
@@ -714,13 +714,13 @@ general for HDDs, the bluestore throttle values are expected to be higher when
 compared to SSDs.
 
 
-Specifying  Max OSD Capacity
-----------------------------
+Override Max IOPS Capacity of an OSD
+------------------------------------
 
-The steps in this section may be performed only if you want to override the
-max osd capacity automatically set during OSD initialization. The option
-``osd_mclock_max_capacity_iops_[hdd, ssd]`` for an OSD can be set by running the
-following command:
+The steps in this section may be performed if you want to override the max IOPS
+capacity of an OSD that was automatically set during its initialization.
+The option ``osd_mclock_max_capacity_iops_[hdd, ssd]`` for an OSD can be
+overridden by running the following command:
```

Comment:
running a command of the following form:

This is subtle but it reinforces that one has to tailor the command vs pasting as-is.